opencodelists/forms.py

```diff
@@ -146,3 +146,36 @@ def save(self, commit=True):
         if commit:
             user.save()
         return user
+
+
+class MembershipCreateForm(forms.Form):
+    email = forms.EmailField(
+        help_text="Enter an email address for an existing user",
+        widget=forms.TextInput(
+            attrs={"class": "form-control", "placeholder": "Email address"}
+        ),
+    )
```

Comment:
Given that users use usernames instead of email addresses to log in, can I suggest using username instead of email address in this form?

Reply:
I thought it was more likely that an admin user would know their email address than username.  Would it make sense to allow them to enter either?

Reply:
>  Would it make sense to allow them to enter either?

Yes, if that's easy!

Reply:
Done :slightly_smiling_face: